### PR TITLE
Hint on how to use not supported DSL.

### PR DESCRIPTION
**Pull request by MecuSorin**

### CODE REVIEW

**Comment by MangelMaxime** [comment]
I would prefer to provide an helper like: https://github.com/fable-compiler/fable-react/blob/master/src/Fable.Recharts/Fable.Recharts.fs#L160

**Comment by chadunit** [comment]
Sorry about the timing, after #58 this needs to use concrete CSSProp.

Should the helper be two arguments or a tuple argument? Two arguments seems preferable but is inconsistent with the other helpers linked as well as the already existing Data helper.

Should the Data helper be a DU static member like this instead of a standalone function?

Should HTMLAttr also have a Custom helper?

**Comment by alfonsogarciacaro** [comment]
Thanks for the review @chadunit. I agree with your comment, actually I had written another one along the same lines but you made me realize I hadn't clicked "Submit review" so it wasn't visible yet :wink:

For consistency, I also think we should use tupled arguments. About using a module function vs a static member, I made `Data` and `Style` module members so it wasn't necessary to qualify them, but `Custom` has more chances of conflicting (especially if we add it also for HTMLAttr and SVGAttr) so making it a static member may prevent problems.

File: src/Fable.React/Fable.Helpers.React.fs [hunk 1]
```diff
@@ -718,6 +718,8 @@ module Props =
         | ZIndex of obj
         | Zoom of obj
         interface ICSSProp
+        /// If you are searching for a way to provide a value not supported by this DSL then use something like: CSSProp.Custom "align-content" "center"
+        static member inline Custom (key: string) (value: obj) : ICSSProp = !!(key, value)
```
[the remark this answers is not in the document]
Comment (MangelMaxime) [reply src/Fable.React/Fable.Helpers.React.fs]: I agree we should favor consistency here.

**Comment by MecuSorin** [comment]
Switched to tupled arguments. 
Regarding the  module function vs static member, we should consider from js output also, not just from F# perspective (to remove clashes, we can name it if needed CSSPropCustom | CustomCSSProp).

**Comment by alfonsogarciacaro** [comment]
Thanks @MecuSorin! Let's merge this and then make the necessary adjustments before releasing the next Fable.React version :+1: In this case, the methods are inlined so there'll no difference in the generated JS. Also, in Fable 2.0 all type methods (both instance and static) will be compiled as module members so choosing one or the other will be mainly a matter of code organization :)

**Pull request merged**